apps/billing/payments/zibal_client.py

```
@@ -145,182 +155,150 @@ def request_payment(
-        try:
-            result = self._make_request('v1/inquiry', payload)
-            return result
-        except Exception as e:
-            logger.error(f"Inquiry failed for track_id {track_id}", exc_info=True)
-            raise ZibalError(f"Inquiry failed: {str(e)}")
-    
+
+        result = self._make_request('v1/inquiry', payload)
+
+        if result.get('result') != self.RESULT_SUCCESS:
+            raise ZibalError(f"Inquiry failed: {self.get_result_message(result.get('result'))}")
```

Ruff (0.14.0)

Avoid specifying long messages outside the exception class

(TRY003)